EssentialCSharp.Web/Services/ICaptchaService.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace EssentialCSharp.Web.Services
+{
+    public interface ICaptchaService
+    {
+        Task<HttpResponseMessage> Verify(string secret, string token, string remoteIp);

```

Comment:
Do we need this overload? It looks like we are only using the other one. 

Do we need both, or should remoteIp be an optional parameter (perhaps of type IPAddress?)?

Reply:
Removed overload, just added a comment instead.